pkg/storaged/filesystem/mismounting.jsx

```diff
@@ -178,9 +178,9 @@ export const MismountAlert = ({ warning, fstab_config, forced_options, backing_b
         else if (type == "mount-on-boot")
             return do_mount();
         else if (type == "no-mount-on-boot")
-            return do_unmount();
+            return do_unmount(old_dir);
         else if (type == "mounted-no-config")
-            return do_unmount();
+            return do_unmount(old_dir || other);
```

Comment:
This should just be `other`, I think. I'll try to make a test that proves this.

Reply:
Yep: https://cockpit-logs.us-east-1.linodeobjects.com/pull-19697-20231205-151550-901259c1-fedora-39-storage/log.html

Reply:
Squashed your test and applied your suggestion to only use `other`.